tuxemon/event/actions/set_party_status.py

```diff
@@ -33,39 +33,50 @@ class SetPartyStatusAction(EventAction):
     character: str
 
     def start(self) -> None:
-        character = get_npc(self.session, self.character)
-        if character is None:
+        char = get_npc(self.session, self.character)
+        if char is None:
             logger.error(f"{self.character} not found")
             return
-        if not character.monsters:
-            logger.error(f"{character.name} has no monsters!")
+        if not char.monsters:
+            logger.error(f"{char.name} has no monsters!")
             return
 
         # parameters
         level_lowest = MAX_LEVEL
         level_highest = 0
         level_average = 0
+        _healthy = 0
+        _lost_hp = 0
 
         # get values
-        for npc_monster in character.monsters:
-            if npc_monster.level < level_lowest:
-                level_lowest = npc_monster.level
-            if npc_monster.level > level_highest:
-                level_highest = npc_monster.level
-            level_average += npc_monster.level
-        level_average = int(round(level_average / len(character.monsters)))
+        for monster in char.monsters:
+            if monster.level < level_lowest:
+                level_lowest = monster.level
+            if monster.level > level_highest:
+                level_highest = monster.level
+            if monster.current_hp == monster.hp:
+                _healthy += 1
+            _lost_hp += monster.hp - monster.current_hp
+            level_average += monster.level
+
+        level_average = int(level_average / len(char.monsters))
```

Comment:
does it make more sense to round the number instead of truncating it?

Reply:
True, I'm going to add the round